README.md

```diff
@@ -78,18 +78,19 @@ Reporting Issues
 
 To report an issue please:
 
-- First post to forum to verify the issue; 
-- Link forum thread to bug tracker ticket and vice-a-versa; 
-- Use the most updated stable or development versions of FreeCAD; 
-- Post version info from eg. `Help > About FreeCAD > Copy to clipboard`; 
+- First post to forum to verify the issue and search the existing [issues](https://github.com/FreeCAD/FreeCAD/issues) for potential duplicates; 
```

Comment:
While we're at it, maybe soften this to "consider posting to the Forum, Discord channel, or Reddit to verify the issue".